### PR TITLE
Frontend-styles

### PR DESCRIPTION
**Pull request by ainarssondors**
-Fixed pdp related product styling
 -Fixed product card padding on mobile/tablet
 -Fixed pdp add to cart button height
 -Added margin for PDP manufacturer
 -Fixed cart page width styling

### CODE REVIEW

File: src/app/component/RelatedProducts/RelatedProducts.style.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     &-List {
         display: grid;
-        grid-template-columns: 1fr 1fr 1fr 1fr;
+        grid-template-columns: 25% 25% 25% 25%;
```
Comment (vladimirsm) [inline src/app/component/RelatedProducts/RelatedProducts.style.scss]: Why you changed to percents here? To shorten you can use `grid-template-columns: repeat(4, 1fr)`

File: src/app/component/ProductDetails/ProductDetails.style.scss [hunk 2]
```diff
@@ -25,6 +25,7 @@
         text-transform: uppercase;
         color: var(--manufacturer-color);
         margin-bottom: 0;
+        margin-left: 0.2rem;
```
Comment (vladimirsm) [inline src/app/component/ProductDetails/ProductDetails.style.scss]: As a shorthand you can use `.2rem;`

File: src/app/component/RelatedProducts/RelatedProducts.style.scss (hunk 1 above)
Comment (alfredsgenkins) [reply src/app/component/RelatedProducts/RelatedProducts.style.scss]: 1fr should be used with caution. Please check tablet reasolution.

Comment (ainarssondors) [reply src/app/component/RelatedProducts/RelatedProducts.style.scss]: It was mentioned in the ticket that fr broke on tablet/mobile. That's why it's now using percentage.

**Comment by alfredsgenkins** [comment]
FIxes #66

**Pull request merged**